web_common/web_common/styles/main.scss

```diff
@@ -452,3 +452,39 @@ td a.fill-td {
   width: 100%;
   height: 100%;
 }
+//Progress bar css
+
+div .bar {
+  display: inline-block;
+  margin: 0;
+}
+
+.textbox {
+  display: none;
+}
+
+.progress_bar {
```

Comment:
Do we need to make these changes specific to the progress bar? Something like `.progress_bar div .bar`?